src/NativeRNDeviceInfoAndroid.ts

```diff
@@ -0,0 +1,154 @@
+import type { TurboModule } from 'react-native';
```

Comment:
Do we really want to split it into 2 files? Since they are almost similar, wouldn't it just be better to add NOOPs in the native code for unsupported methods? Wdyt?

Reply:
That would mean adding 70 NOOPs to iOS. I think it makes it too difficult to read.

Reply:
Wow, that's a lot. Ok so let's keep it this way for now.